### PR TITLE
Add support for user defined notification sounds

### PR DESCRIPTION
**Pull request by f3bruary**
I'm no coder, but the code looked simple enough to give it a shot. This pull requests adds the ability to define your own ringtone.

I personally like 'cashregister' for new orders :)

### CODE REVIEW

File: classes/class-pushover-api.php [hunk 1]
```diff
@@ -184,7 +201,7 @@ function send() {
 		$request = array(
 						'token' 	=> $this->site_api,
 						'user'  	=> $this->user_api,
-						'title'     => $this->title,
+						'title'     	=> $this->title,
 						'message'	=> $this->message,
 						'url' 		=> $this->url,
```
Comment (pogla) [inline classes/class-pushover-api.php]: You need to add

```
'sound'   => $this->sound,
```

in the request array.

File: classes/class-wc-pushover.php [hunk 2]
```diff
@@ -130,6 +131,16 @@ function init_form_fields() {
...
+			'sound' => array(
+				'title'       => __( 'Notification Sound', 'wc_pushover' ),
+				'description' => sprintf(
+					'%s <a href="https://pushover.net/api#sounds" target="_blank">%s</a>',
+					__( 'Select from', 'wc_pushover' ),
+					__( 'here', 'wc_pushover' )
+				),
+				'type'        => 'text',
+				'default'     => '',
+			)
```
Comment (pogla) [inline classes/class-wc-pushover.php]: This is causing an error. You need to add a comma after the ")".

File: classes/class-wc-pushover.php [hunk 3]
```diff
@@ -558,7 +570,8 @@ function send_notification( $args ) {
 							"\nTitle: ". $args['title'] .
 							"\nMessage: ". $args['message'] .
 							"\nURL: " . $args['url'] .
-						  "\nPriority: " . $this->priority );
+						  	"\nPriority: " . $this->priority ) .
```
Comment (pogla) [inline classes/class-wc-pushover.php]: This is causing an error. You need to remove the ")".

File: classes/class-pushover-api.php (hunk 1 above)
Comment (f3bruary) [reply classes/class-pushover-api.php]: Thanks, I added it. I wasn't sure because there also wasn't a Priority in this array.

**Comment by f3bruary** [comment]
I made the changes. I there anything else I need to do, or can this be merged now ?

**Comment by pogla** [comment]
@f3bruary This looks good now. I tested it and it's working great. 

@growdev I am merging this to master now.

**Pull request merged**